Components/buttons/button.css

```diff
@@ -1,170 +1,189 @@
+.btn.btn-icon:hover,
+.btn.btn-link:hover {
+    color: var(--text-active-color);
+    background-color: transparent;
+}
+
+/* General outlined button hover when no theme specified */
+.btn.btn-outline:hover {
+    color: var(--bg-color);
+}
+
+/* Button with primary background color */
 .btn-primary {
     background-color: var(--primary-color);
     border: 2px solid var(--primary-color);
 }
 
-.btn-accent {
-    background-color: var(--accent-color);
-    border: 2px solid var(--accent-color);
+/* Primary button hover */
+.btn-primary:hover {
+    background-color: var(--primary-active-color);
+    border-color: var(--primary-active-color);
 }
 
-.btn-outline {
-    background-color: transparent;
+/* Button with secondary background color */
+.btn-secondary {
+    background-color: var(--secondary-color);
+    border: 2px solid var(--secondary-color);
 }
 
-.btn-outline.btn-primary {
-    color: var(--primary-color); 
+/* Secondary button hover */
+.btn-secondary:hover {
+    background-color: var(--secondary-active-color);
+    border-color: var(--secondary-active-color);
 }
 
-.btn-outline.btn-accent {
-    color: var(--accent-color); 
+/* Primary outline button */
+.btn-outline.btn-primary {
+    color: var(--primary-color); 
 }
 
-.btn-icon {
-    background-color: transparent;
-    border: none;
-    font-size: var(--text-md);
-    min-width: max-content;
-    min-width: max-content;
+/* Secondary outline button */
+.btn-outline.btn-secondary {
+    color: var(--secondary-color); 
 }
 
+/* Primary button icon */
 .btn-icon.btn-primary {
     color: var(--primary-color);
 }
 
-.btn-icon.btn-accent {
-    color: var(--accent-color);
+.btn-primary.btn-icon:hover,
+.btn-primary.btn-link:hover  {
+    color: var(--primary-active-color);
 }
 
-.btn-link {
-    background-color: transparent;
-    border: none;
-    border-radius: 0;
-    color: var(--text-color);
-    border-bottom: 2px solid var(--text-color);
-    font-weight: var(--weight-sbold);
-    padding: 0;
-    min-width: max-content;
-    min-height: max-content;
-    -webkit-border-radius: 0;
-    -moz-border-radius: 0;
-    -ms-border-radius: 0;
-    -o-border-radius: 0;
+/* Secondary button icon */
+.btn-icon.btn-secondary {
+    color: var(--secondary-color);
 }
 
-.btn-text-icon {
-    display: flex; 
-    align-items: center;
-    justify-content: center;
-    gap: var(--space-0-5);
+.btn-secondary.btn-icon:hover,
+.btn-secondary.btn-link:hover  {
+    color: var(--secondary-active-color);
 }
 
+/* Floating button fixed to the right bottom */
 .btn-floating {
     border-radius: var(--br-50);
     -webkit-border-radius: var(--br-50);
     -moz-border-radius: var(--br-50);
     -ms-border-radius: var(--br-50);
     -o-border-radius: var(--br-50);
+    border: none;
+
     width: var(--space-3);
     height: var(--space-3);
     min-width: var(--space-3);
     min-height: var(--space-3);
+
     display: flex;
```

Comment:
Spacings ? If it's for the sake of visual sectioning for reviewers then it's okay!

Reply:
Yes, it is for that purpose. I have sectioned properties together. E.g. display, font, flex, size, border, etc. Makes it easier to debug, add, and review.